src/main/webapp/search-interview.js

```diff
@@ -18,42 +18,67 @@ function onSearchInterviewLoad() {
   loginInfo.then(getUserOrRedirectRegistration);
 }
 
-// Should query Datastore for appropriate interviews and render them on the
+// Queries Datastore for available interview times and renders them on the
 // page.
 function loadInterviews() {
-  // TODO: fetch and get all interviews instead of unhiding something hidden
   const searchResultsDiv = document.getElementById("search-results");
   searchResultsDiv.removeAttribute("hidden");
+
+  fetch(`/load-interviews?timeZoneOffset=${browserTimezoneOffset()}`)
+    .then(response => response.text())
```

Comment:
Fix indentation (`.then`s should be aligned here)

Reply:
Done